### ISSUE TITLE
Option to globally disable AUX fan - which causes adhesion issues

### ISSUE THREAD

**Issue by tomasz90** [bambulab/BambuStudio#4640]
**Is your feature request related to a problem? Please describe.**
AUX fan cause to lift my prints, often ruin them completly. For instance I have naver had issues with PLA printing on different printer. On P1S if AUX is on, PLA is always warping.

**Describe the solution you'd like**
 I would like to be able to diable it globally.

**Describe alternatives you've considered**

**Additional context**

**Comment by RicVilla78** [comment]
you can disable in all you PLA profiles directly under cooling.
![image](https://github.com/user-attachments/assets/b24b05cf-2b77-47c0-b213-f5ad8d22fd30)

**Comment by tomasz90** [comment]
> you can disable in all you PLA profiles directly under cooling. ![image](https://private-user-images.githubusercontent.com/53941830/358488857-b24b05cf-2b77-47c0-b213-f5ad8d22fd30.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjM3OTEwMzcsIm5iZiI6MTcyMzc5MDczNywicGF0aCI6Ii81Mzk0MTgzMC8zNTg0ODg4NTctYjI0YjA1Y2YtMmI3Ny00N2MwLWIyMTMtZjVhZDhkMjJmZDMwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODE2VDA2NDUzN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWFhYThiMzY2OTVlNjJiNTY2ZGJjOTU4OGQxMDAwNjRmM2ZmY2IyYTUzOWQ0MzI5YWRlYWY2MTYzMTUwMTUwNGUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.T4U_zdufo_pOshV3RmTEVdV9cL3GhvrN6CtJdvvI-YA)

I use AMS, I would need to make copy of every filament preset and remember to use them... VERY INCONVENIENT.

**Comment by RicVilla78** [comment]
I use AMS all the time….you are supposed to have a profile for each filament…if you don’t, you are not using the system as intended. If you only use one profile for all your PLAs then deactivate it there and use that one. I don’t understand what your problem is honestly. Putting a global switch to deactivate it makes no sense and specifically for PLA the aux fan is very handy to print fast, warping is just a symptom of another problem you may have.

**Comment by tomasz90** [comment]
From your perspective, I need to create copy of every filament I bought from bambu and overwrite this setting???? 
So what is the point of the NFC stickers????

**Comment by RicVilla78** [comment]
Yes and no, the purpose of the RFID is only to know the type, color, nozzle and bed temperatures and how much filament is it left. I doesnt store a full filament profile. Also no if the fan will be on or off, that is stored in the filament profile.

I you have 3 types of PLA, copy those and uncheck the Aux fan box and leave the original profile as "backup" in case something goes wrong. Probably you can put in in some of the G-code but I am not sure in which order does Bambu Studio processes all the different Start G-codes (Printer, Filament, Process...probably in that order).

**Comment by tomasz90** [comment]
Not only me sees it as an issue ;/
Yesterday AUX fan broken my 10 hours print, because it turned on while printing PVA interface and ruin PLA adhesion... I had different profile for PLA, but NOT PVA. This is ridicolous to create every different profile for bambu stock filament ;/

<img width="989" alt="Screenshot 2024-08-17 at 11 49 28" src="https://github.com/user-attachments/assets/734e65c2-4e66-4270-8877-66d21270b787">

**Comment by tomasz90** [comment]
This is joke
<img width="1045" alt="Screenshot 2024-08-17 at 11 53 48" src="https://github.com/user-attachments/assets/cde443f3-5763-4edc-8c3b-b76a3876de05">

**Comment by RicVilla78** [comment]
Although I still don’t think your proposal makes sense, I have something it may help you. I don’t have a p1S so I don’t know if that works, but the x1c it can be deactivated under the maintenance menu. May be in the p1s is also available.

![IMG_2150](https://github.com/user-attachments/assets/84efd3c5-bb70-4178-b1d3-996a10d23f86)

**Comment by hippoduck** [comment]
The aux fan is the bain of my life. I print lots of small items, and they always fail around the aux fan. It's just cooling the part and the plate too much on that side. 

I created filament profiles but it's easy to forget to change it, and wake up to a failed print. 

If it really helps, then why is it only on one side? Surely that means uneven cooling. I'd love a setting to disable it on my P1S, but may just have to pull the fan cable.

**Comment by tomasz90** [comment]
Another print ruined by this f*cking fan... I will not wait for bambu fix...
![IMG_20240904_104923267](https://github.com/user-attachments/assets/c7ee1ccb-6108-4693-b78a-19d7accc9bea)

**Comment by hippoduck** [comment]
@tomasz90 Assuming you made that connector. Mine has the greyscale cable all the way to the fan.

**Comment by lar3ry** [comment]
I would REALLY like a way to either set a new profile as default or the ability to change a Bambua Lab supplied profile and save i under the same name.

**Comment by tomasz90** [comment]
Exactly... Change the default settings under the same name, with some reset button - it would do the trick.

**Comment by sol3uk** [comment]
I'm really surprised this isn't a setting. I'm constantly having warping issues with PLA on only the side that the Aux fan is blasting away at the print. When I turn it off, it prints fine. But making overriding profiles for each filament is such a non solution.

**Comment by tomasz90** [comment]
@sol3uk just cut off the wire, I doubt that bambu will fix this in the near future..

**Comment by 3dfdm** [comment]
A simple "disable aux fan" option that overridesall filament profiles wouldbe good.
I have seen a very lot of aux fan issues on multiple social media platforms that would be resolved ifonly we could permanently disable it.
Whilst the aux fan may have it's use (I'm not 100% sure where that is) there is also the need by many users to not have one.

**Comment by houmark** [comment]
> A simple "disable aux fan" option that overridesall filament profiles wouldbe good. I have seen a very lot of aux fan issues on multiple social media platforms that would be resolved ifonly we could permanently disable it. Whilst the aux fan may have it's use (I'm not 100% sure where that is) there is also the need by many users to not have one.

Yes a global setting would be very nice. With 3 AMS units and 12 colors loaded, it's pretty time consuming to change all the colors loaded to a different filament profile, where the only change from the original profile is setting the AUX to 0%.

I have run without the AUX fan for months now and was considering to physically disconnect it. Then I printed two different funnels the other day and had really, really bad surfaces both on the outside and the inside on both models which were quite different in design actually.

After considering options and trying a few different test prints I realized that the hot end comes around to the next layer so fast on these types of models that the prior layer has not yet stabilized or cooled down enough. The result is that the layers are bending outward (on the outside and inward on the inside) and makes the surface really bad.

I turned back on the AUX fan to the standard 70% and the funnel printed with near perfect surfaces. Night and day difference.

That made me consider if I should try the AUX fan again on a more constant basis, even though with Sunlu PLA+ (my main filament) I had a lot of warping on models in the past, even using SuperTack plate, which have given me the very best sticking of any plate I used up until now (to the point where some prints can be hard to get off).

Well, the first print I did warped a bit in two of the four corners, so I guess I am back to turning it off 95% of the time and only back on when doing smaller (round) pieces where the toolhead is moving upward in layers very fast.
